run.py

```diff
@@ -0,0 +1,36 @@
+
+import argparse
+import logging.config
+logging.config.fileConfig("config/logging/local.conf")
+logger = logging.getLogger("dear-genie")
+
+from src.load_data import load_data
+from src.model import create_sqlite_db, create_rds_db
+from config import BUCKET_NAME, SQLALCHEMY_DATABASE_URI, DATABASE_NAME
+
+if __name__ == '__main__':
+
+    parser = argparse.ArgumentParser(description="Data processes")
+    subparsers = parser.add_subparsers()
+
+    sub_process = subparsers.add_parser('loadS3')
+    sub_process.add_argument("--bucket", type=str, default=BUCKET_NAME, help="Bucket to be copied to")
```

Comment:
A suggestion is to not have a default bucket defined in config and make this argument mandatory. The reason for this is that if someone else is using your code, they will definitely have to put in the information of their S3 bucket.